repos/system_upgrade/el8toel9/actors/satellite_upgrade_facts/actor.py

```diff
@@ -0,0 +1,75 @@
+from leapp.actors import Actor
+from leapp.libraries.common.config import architecture
+from leapp.libraries.common.rpms import has_package
+from leapp.models import (
+    InstalledRPM,
+    RepositoriesSetupTasks,
+    RpmTransactionTasks,
+    SatelliteFacts,
+    SatellitePostgresqlFacts
+)
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+SATELLITE_VERSION = '6.99'
```

Comment:
obviously will need to be updated to the right Satellite version, once we know what it will be.

Reply:
Could you please add a TODO here then? If this can get in before the right version is there?

Reply:
Actually, we can do better, we can compute the correct name for the el9 repo based on the list of enabled el8 repos!
Will do on Monday.
(Ideas you have when doing dishes and listening to 90ies pop music 😅)